### DISCUSSION TITLE
Liquid fixity on inclined plane

### DISCUSSION THREAD

**Discussion by ykevin09** [Anura3D/Anura3D_OpenSource#34] [Q&A] [answered]
Hi.

I want to simulate a slope failure problem, which requires liquid boundary conditions on the inclined plane: zero liquid velocity perpendicular to the slope (impermeable). Could you please help how to achieve that? I fixed the vertical and horizontal liquid velocities in the attached figure but feel they are not realistic.

<img width="655" alt="image" src="https://user-images.githubusercontent.com/7458589/186697655-df1f840c-a996-403b-b273-06f88baeaaca.png">

**Comment by murphy-john** [comment, marked as answer]
At this time, Anura3D only supports fixities in the x, y, and z directions.  Fixities at an angle or on a slope are not possible but we'd like to add them. 

One work around may be to add a layer of relatively low permeability soil above your loose sand layer. You could make the low permeability layer weak and low density. But you'll have to consider how those assumptions effect your model.

**Reply by ykevin09** [reply]
Thanks for the reply and the suggestion.
